### PR TITLE
Feature/1008 shell docs

### PR DESCRIPTION
**Pull request by xak**
Closes sap/fundamental#1008

Implements `shell` layout containers to replace `ui`; implements `shellbar` to replace `global-nav`. Also updates all demo pages to use the new shellbar and layout containers.

> #1055 must be completed before release

#### Test

* https://deploy-preview-1057--fundamental.netlify.com/layouts/index.html
* https://deploy-preview-1057--fundamental.netlify.com/layouts/application-layout.html
* https://deploy-preview-1057--fundamental.netlify.com/components/shellbar.html
* https://deploy-preview-1057--fundamental.netlify.com/demo-pages/index.html
https://deploy-preview-1057--fundamental.netlify.com/demo-pages/full-width-app-demo.html

#### Changelog

**New**

* Shellbar added conforming to Fiori 3 specifications

**Changed**

* High-level layout containers have been updated to use `shell`
* App-level `app__sidebar` container is now `app__naviagtion` with `--horizontal` and `--vertical` orientations 

**Removed**

* Layout `ui` and `global-nav` will be deprecated in a future release

### CODE REVIEW

**Comment by joseegm** [comment]
Test Version for this PR was deployed

Built with commit b7544e934d52f1aebc1dffe5a864dd39aa8d3367

https://deploy-preview-1057--fundamental.netlify.com

File: docs/scss/_docs-display-component.scss [hunk 1]
```diff
@@ -72,32 +72,70 @@
...
+          .fd-app {
+              width: 100%;
+              max-width: 100%;
+              min-height: 410px;
+              max-height: 410px;
+              //position: relative;
+          }
+
+
+      // .fd-shell__app,
```
Comment (ckoutsiaris) [inline docs/scss/_docs-display-component.scss]: Can this be removed?

File: docs/scss/_docs-display-component.scss [hunk 2]
```diff
@@ -72,32 +72,70 @@
...
             background-color: fd-color(neutral, 3);
         }
 
-        .fd-app__sidebar {
-            width:250px;
+        .fd-app__navigation {
+
             background-color: fd-color(background, 2);
-            padding-top:100px;
+            //padding-top:100px;
```
Comment (ckoutsiaris) [inline docs/scss/_docs-display-component.scss]: Can this be removed?

File: scss/components/shellbar.scss [hunk 3]
```diff
@@ -162,20 +163,20 @@ $block: #{$fd-namespace}-shellbar;
...
-            display: none;
+            //display: none;
         }
         @include fd-screen(m) {
-            display: none;
+            //display: none;
         }
         @include fd-screen(l) {
-            display: inline-block;
+            //display: inline-block;
```
Comment (ckoutsiaris) [inline scss/components/shellbar.scss]: Can we remove the commented display:none ?

File: test/templates/product-menu/component.njk [hunk 4]
```diff
@@ -33,13 +33,14 @@ product_menu:
   }}
 </div>
 {%- else %}
-{{ product_menu_logo(logo=brand.logo, name=brand.name) }} <span class="fd-shellbar__title">{{ product.title }}</span>
+{# {{ product_menu_logo(logo=brand.logo, name=brand.name) }}  #}<span class="fd-shellbar__title">{{ product.title }}</span>
```
Comment (ckoutsiaris) [inline test/templates/product-menu/component.njk]: commented?

File: docs/scss/_docs-display-component.scss (hunk 1 above)
Comment (xak) [reply docs/scss/_docs-display-component.scss]: Done

File: docs/scss/_docs-display-component.scss [hunk 5]
```diff
@@ -72,32 +72,70 @@
...
+
+
+        .fd-shell__app,
+          .fd-shell,
+          .fd-app {
+              width: 100%;
+              max-width: 100%;
+              min-height: 410px;
+              max-height: 410px;
+              //position: relative;
```
[the remark this answers is not in the document]
Comment (xak) [reply docs/scss/_docs-display-component.scss]: Done

File: scss/components/shellbar.scss (hunk 3 above)
Comment (xak) [reply scss/components/shellbar.scss]: Done. Removed the whole block.

File: docs/scss/_docs-display-component.scss (hunk 2 above)
Comment (xak) [reply docs/scss/_docs-display-component.scss]: Done

File: test/templates/product-menu/component.njk (hunk 4 above)
Comment (xak) [reply test/templates/product-menu/component.njk]: Done. I revisited the duplicate logo and opted to remove it.

File: test/templates/shellbar/component.njk [hunk 6]
```diff
@@ -196,7 +196,7 @@ shellbar:
             </div>
         {%- endfor %}
         <div class="fd-shellbar__action fd-shellbar__action--collapse">
-          <!--{{ shell_actions_collapse(actions=collapsed_items) }}-->
+          {# {{ shell_actions_collapse(actions=collapsed_items) }} #}
```
Comment (ckoutsiaris) [inline test/templates/shellbar/component.njk]: Maybe remove this one too?

Comment (saadmhybris) [reply test/templates/shellbar/component.njk]: fixed

**Pull request merged**